app/models/framework/definition/CM_OSG_05_3565.rb

```diff
@@ -20,10 +20,10 @@ class Invoice < EntryData
         field 'Item Code', :string, exports_to: 'ProductCode'
         field 'Manufacturers Product Code', :string, exports_to: 'Additional1'
         field 'Unit Quantity', :string, exports_to: 'Additional2'
-        field 'Price per Item', :string, exports_to: 'UnitPrice', ingested_numericality: true, allow_nil: true
+        field 'Price per Item', :string, exports_to: 'UnitPrice', ingested_numericality: true
```

Comment:
Can't verify against template, as the file doesn't have any comments about what's mandatory.

Reply:
@CristinaRO Generally speaking orange headers are mandatory and blue are optional. CM/OSG/05/3565 is being expired as it's very old and therefore the template isn't as descriptive as the newer ones.

Reply:
Confirmed with Malcolm. We should always be validating the unit price as it's one of the core fields.